airflow/www/views.py

```diff
@@ -353,6 +356,7 @@ def get_int_arg(value, default=0):
                                            status=arg_status_filter if arg_status_filter else None),
             num_runs=num_runs,
             tags=tags,
+            state_color=state_color_mapping,
```

Comment:
Hi. Would you like to add a test that will check if the correct value was passed to the template? A method has been added today that makes it easy. I think it's worth gradually extending this idea to other tests to increase the confidence of the Web UI tests.
https://github.com/apache/airflow/pull/8505

Reply:
Hi @mik-laj , thanks for the suggestion.

I have done an update by adding an assert in an existing test, which should suffice the purpose here.

Let me know if you have any other suggestion? Cheers.

Reply:
Hi @mik-laj , I have further updated the UI test code using the method you shared. Please take a look when you get time.

Two tests are failing, https://travis-ci.org/github/apache/airflow/jobs/684134650#L16198 and https://github.com/apache/airflow/pull/8742/checks?check_run_id=652054298 . But both seem interim and irrelevant to the changes of this PR